feat: Support for experiment names (#54)
neptune-ai/neptune-fetcher: README.md

__Parameters:__

| Name                | Type              | Default | Description                                                                                                                      |
|---------------------|-------------------|---------|----------------------------------------------------------------------------------------------------------------------------------|

> | `read_only_project` | `ReadOnlyProject` | -       | Source project from which run will be fetched.                                                                                   |

Reviewer: ```suggestion
| `read_only_project` | `ReadOnlyProject` | -       | Project from which the run is fetched.                                                                                   |
```
